### ISSUE TITLE
Move NMR schema to entrypoint

### ISSUE THREAD

**Issue by ndaelman-hu** [nomad-coe/electronic-parsers#266]
Part of the NMR schema is made available via `electronicparsers/utils/margres_workflow.py`, whereas it only serves 2 parsers at most.
Connect this entrypoint to a single parser or make it standalone.

**Comment by Sathya-S3** [comment]
Hi @ndaelman-hu,

Currently, both the schema and magres parser have been moved into a dedicated magres parser for nomad https://github.com/CCP-NC/nomad-parser-magres. In an attempt to keep the schema separate so that it can continue growing with more NMR contributions from the community, I'm creating a schema repository for NMR in https://github.com/CCP-NC organisation page. I'm using the cookie cutter tool in the nomad repository template to do this. 

We can hopefully join efforts there and avoid duplication.

Best regards,
Sathya.

**Comment by ndaelman-hu** [comment]
@Sathya-S3 thank you for the heads-up! I originally opened this issue more so from the POV of dependency management. No migration has actually been slated yet, and we'll of course let you know beforehand (likely when migrating to `data`).

As for the partitioning and hosting of the NMR project, I think that @JFRudzinski and @andreaa93 have been following this up better with you, so I'll defer to them.

**Comment by JFRudzinski** [comment]
Exactly, thanks @ndaelman-hu. @Sathya-S3 I guess Andrea and Cecilia will discuss with you more in your upcoming meeting. But the basic plan is for Cecila to work on a few extensions within our existing (run) schema, trying to not duplicate what you have done, or possibly incorporating some of your efforts already. The reason for this approach is mainly due to time constraints. 

Later this year we will migrate all of our parsers to the new (data) schema. At that point, we will approach you to align your efforts with Cecila's. At that point we can establish a clearer protocol for management/extension/alignment.
